xml/security_ldap.xml

```diff
@@ -784,7 +784,95 @@ systemctl start sssd</screen>
  </sect2>
  </sect1>
 
-<sect1 xml:id="sec-security-ldap-yast-usergr">
+<sect1 xml:id="sec-security-ldap-server-yast">
+ <title>Setting Up a &ds389; with &yast;</title>
+  <para>
+   You can use &yast; to quickly create a very basic setup of the &ds389;.
+  </para>
+ <sect2 xml:id="sec-security-ldap-server-yast-create">
+  <title>Creating an &ds389; Instance with &yast;</title>
+  <procedure>
+   <step>
+   <para>
+    In &yast;, click <menuchoice>
+    <guimenu>Network Services</guimenu>
+    <guimenu>Create New Directory Server</guimenu>
+   </menuchoice>. Alternatively, start the module from command line with
+   <command>yast2 ldap-server</command>.
+   </para>
+   <para>
+    In the window that opens, you need to fill in all options.
+   </para>
+  </step>
+  <step>
+   <para>
+    Enter a <guimenu>Fully qualified domain name</guimenu> to use.
+   </para>
+  </step>
+  <step>
+   <para>
+    Enter the <guimenu>Directory server instance name</guimenu>.
+   </para>
+  </step>
+  <step>
+   <para>
+    Enter the <guimenu>Directory suffix</guimenu>.
+   </para>
+  </step>
+  <step>
+   <para>
+    Enter the <guimenu>Directory manager DN</guimenu>.
+   </para>
+  </step>
+  <step>
+   <para>
+    Enter the password for the directory manager and repeat it in the next text
+    box.
+   </para>
+  </step>
+  <step>
+   <para>
+    Enter the path to the <guimenu>Server TLS certificate authority in PEM format</guimenu>.
+    <remark>taroth 2020-01-20: @firstyear, where does the certificate need to be placed? 
+    which path and certificate name to use as an example to get a useful screenshot?</remark>
+   </para>
+  </step>
+  <step>
+   <para>
+    Enter the path to the <guimenu>Server TLS certificate and key in PKCS12 format</guimenu>.
+    <remark>taroth 2020-01-20: @firstyear, where does the certificate need to be placed? 
+    which path and certificate name to use as an example to get a useful screenshot?</remark>
+   </para>
+   <informalfigure>
+    <mediaobject>
+     <imageobject role="fo">
+      <imagedata fileref="yast2-ldap-server.png" width="95%" format="PNG"></imagedata>
+     </imageobject>
+     <imageobject role="html">
+      <imagedata fileref="yast2-ldap-server.png" width="90%" format="PNG"></imagedata>
+     </imageobject>
+    </mediaobject>
+   </informalfigure>
+  </step>
+  <step>
+   <para>
+    If all setting are according to your wishes, confirm with <guimenu>OK</guimenu>.
+   </para>
+  </step>
+ </procedure>
+ <para>
+  <remark>taroth 2020-02-20: @firstyear: does another window follow if I click 'OK'?
+   I could not test it  as I do not have test certificates (and one cannot click
+   'OK' without filling all fields, which makes sense)</remark>
+  &yast; creates a directory server instance with the settings you have specified.
+  <remark>taroth 2020-01-20: @firstyear: what happens in the background?
```

Comment:
Uhhhhh .... some things can be changed, some can't? I think instance name can never be changed post install. Suffix can be changed, but it's a big effort. Everything else is easy to change.